js_functions/task4.js

```diff
@@ -0,0 +1,24 @@
+function divide (numerator, denominator){
+    try {
+        let action = numerator/denominator;
+        console.log(action);
+        if (denominator === 0 || typeof numerator != "number" || typeof denominator != "number")
```

Comment:
for better user experience you should divide this checks. In first one check if all arguments are numbers, in second one check if denominator is not 0